csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/Dependencies.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Semmle.Extraction.CSharp.DependencyFetching
+{
+    /// <summary>
+    /// Container class for dependencies found in the assets file.
+    /// </summary>
+    internal class Dependencies
```

Comment:
Minor: how about changing this to `DependencyCollection` or `DependencyContainer` to avoid plurals in class names.

Reply:
Yes, that is a good idea!